src/StaticFAQ/ArticleFeedback/ScreenInput.js

```diff
@@ -0,0 +1,107 @@
+// @flow
+
+import * as React from 'react';
+import css from 'styled-jsx/css';
+import { Close } from '@kiwicom/orbit-components/lib/icons';
+
+import { Box, ContactPageLink } from '../../common';
+import createComment from '../../mutations/CreateCommentMutation';
+import screensList from './screensList';
+
+type Props = {|
+  changeScreen: (nextScreen: string) => void,
+  articleId: string,
+|};
+type State = {|
+  comment: string,
+|};
+
+const style = css`
+  div.title {
+    color: #171b1e;
+    font-size: 16px;
+    margin-bottom: 8px;
+  }
+  div.question {
+    font-size: 14px;
+    color: #46515e;
+    margin-bottom: 4px;
+  }
+  div.input-area textarea {
+    width: 352px;
+    height: 72px;
+    border-radius: 3px;
+    background-color: #ffffff;
+    border: solid 1px #bac7d5;
+  }
+  div.button button {
+    font-weight: bold;
+    font-size: 14px;
+    width: 100px;
+    height: 44px;
+    border-radius: 3px;
+    border: none;
+    background-color: #00a991;
+    color: #ffffff;
+    cursor: pointer;
+  }
+  div.button {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 19px;
+  }
+  div.close-icon {
+    position: absolute;
+    top: 8px;
+    right: 8px;
+  }
+  div.input-area textarea {
+    padding: 12px 16px;
+  }
+`;
+
+class ScreenInput extends React.Component<Props, State> {
+  state = {
+    comment: '',
+  };
+  handleChange = (e: SyntheticInputEvent<HTMLInputElement>) => {
+    this.setState({ comment: e.target.value });
+  };
+  handleSubmit = (e: SyntheticEvent<HTMLFormElement>) => {
+    const { changeScreen, articleId } = this.props;
+    e.preventDefault();
+    createComment(articleId, this.state.comment, () =>
+      changeScreen(screensList.THANK_YOU),
+    );
+  };
+  render() {
+    return (
+      <Box
+        border="none"
+        padding="40px 24px 24px 24px"
+        borderRadius="4px"
+        backgroundColor="#f5f7f9"
+      >
+        <form onSubmit={this.handleSubmit}>
+          <div className="close-icon">
+            <Close fill="#bac7d5" height="12" />
```

Comment:
what happens if I click on close? should the feedback box hide?

Reply:
Great point, completely forgot. I will make it go back to the initial screen for now. Does that makes sense to you?

Reply:
It makes sense 👍